src/main/webapp/components/workspace/workspace.js

```diff
@@ -82,12 +85,24 @@ const queryToSearch = query => {
   }
 }
 
+const addFiltersToQuery = (query, filters) => {
+  const { filterTree } = query
+
+  const searchTree = !filterTree ? defaultQuery : filterTree
```

Comment:
Sometimes when changing search queries, the filter tree is null. I assume this is when a default query is created and never changed.